token/src/main/thrift/io_warp10_quasar_token_thrift_data.thrift

```diff
@@ -139,6 +139,11 @@ struct ReadToken {
     13:optional map<string,string> attributes,
 
     /**
+     * Token labels
```

Comment:
These are not token labels but extra labels which will be used  for selecting GTS. The comment should reflect that.